client/components/search/index.jsx

```diff
@@ -85,6 +86,8 @@ const Search = React.createClass( {
 			onSearchOpen: noop,
 			onSearchClose: noop,
 			onKeyDown: noop,
+			onClick: noop,
+			overlayStyling: noop,
```

Comment:
The default `noop` was deliberately removed in #8580 as an optimisation so that we [only render the overlay](https://github.com/Automattic/wp-calypso/pull/8282/files#diff-72df316033d6ba1a0229a34c28b9890eR351) if a styling function is passed in as a prop. It might be best to set it to something falsy here with a comment to say why.